src/libraries/Common/src/System/IO/Compression/ZLibNative.cs

```diff
@@ -64,51 +66,53 @@ public enum CompressionStrategy : int
         }
 
         /// <summary>
-        /// In version 1.2.3, ZLib provides on the <code>Deflated</code>-<code>CompressionMethod</code>.
+        /// In version 2.2.1, zlib-ng provides only the <see cref="Deflated" /> <see cref="CompressionMethod" />.
```

Comment:
Good catch. I assume zlib-ng 2.2.1 is the _minimum_ version where this restriction was added?

Reply:
Thanks. I'm describing a lack of support for anything besides Deflated, so the restriction's been present from the outset. The only reason I've picked v2.2.1 is that it's the one we've used at time of writing.

Reply:
Ah okay. That's alright too.